packages/core/src/accessanalyzer/vue/iamPolicyChecks.vue

```diff
@@ -15,10 +15,10 @@
                         <p>Install Python 3.6+</p>
                     </li>
                     <li>
-                        <code> pip install cfn-policy-validator==0.0.31 </code>
+                        <code> pip install cfn-policy-validator==0.0.32 </code>
```

Comment:
could be >=0.0.32

Reply:
We can pin this version for now, in case there are any breaking changes to the CLI tool. Newer versions should bump this if needed.